app/forms/umrdr/forms/work_form.rb

```diff
@@ -31,5 +29,14 @@ def rights
         @model.rights.first
     end
 
+    # You like danger?! Because you better pass in a hash with the correct keys. 
+    # Can be alleviated when form coverage attribute is:
+    # `coverages: [{begin: {:year, :month, :day}, end: {:year, :month, :day}}, ... ]`
+    def merge_date_coverage_attributes!(hsh)
+      hsh ||= {}
+      hsh.stringify_keys!
+      @attributes.merge! hsh
+    end
```

Comment:
This is the function that belongs to form instances that does the merging of the date coverage parameters with the form attributes.  Is there a more concise and clear way of accomplishing this?